port/unix/omrsysinfo.c

```diff
@@ -7934,6 +7934,17 @@ omrsysinfo_get_process_name(struct OMRPortLibrary *portLibrary, uintptr_t pid)
 			return exename;
 		}
 	}
+#elif defined(OSX) /* defined(LINUX) && !defined(OMRZTPF) */
+	char exebuf[PROC_PIDPATHINFO_MAXSIZE];
+	int pidpathrc = proc_pidpath((int)pid, exebuf, sizeof(exebuf));
+	if (pidpathrc > 0) {
+		uintptr_t exelen = (uintptr_t)pidpathrc + 1;
```

Comment:
On second thought (after the names have changed), the `+ 1` doesn't belong here for consistency with the meaning of `exelen` for Linux above.

Reply:
ok, I changed the variable name to alloclen.